tests/SimpleStringTest.cpp

```diff
@@ -257,6 +257,12 @@ TEST(SimpleString, countTogether)
     LONGS_EQUAL(4, str.count("ha"));
 }
 
+TEST(SimpleString, countEmptyString)
+{
+    SimpleString str("hahahaha");
+    LONGS_EQUAL(1, str.count(""));
```

Comment:
This lead to a noticable delay (30 seconds or so), as well as the replace function not working properly for empty strings (the else part was never executed).